test/performance/observed-concurrency/reporter.lua

```diff
@@ -14,7 +14,16 @@
 
 -- parses a response as returned by the system under test
 function parse_response(res)
+  if (res == nil or res == '') then
+    print('ERROR: Empty response from the test.')
+    os.exit()
+  end
   local split_at, _ = res:find(',', 0)
+  if (split_at == nil or split_at == '') then
+    print('ERROR: Split_at is nil.')
```

Comment:
I commented about the error message and you replied it was updated, but I don't see the changes...